### PR TITLE
Allow bypass elasticsearch certificate validation

### PR DESCRIPTION
**Pull request by horacimacias**
Resubmitting on a single commit with all the suggested changes

### CODE REVIEW

**Comment by tsaikd** [comment]
Please rebase to check CI work well.

**Comment by horacimacias** [comment]
I've tried to rebase now, can you please check if that's what you asked for?

**Comment by tengattack** [comment]
It seems not a rebase git operation? After rebasing, this PR should only contain one commit.

**Comment by horacimacias** [comment]
sorry.... let me try again

**Comment by tsaikd** [comment]
This MR is still not good rebase state, your commit `Allow bypass elasticsearch certificate validation` should be after 1159768fa91586b6caecaf18d7942f4cdcb4109f. Your MR should not container my commits.

You can use `gitk --all` to visualize commit history.
image
